### PR TITLE
Add more jobs to kettle's garbage job list

### PR DESCRIPTION
**Pull request by MushuEE**
#18480

This should exclude some more jobs from Triage board / BQ
... I really now want to rewrite this in Golang 

/area kettle
/cc @BenTheElder @michaelkolber @cjwagner 

Also will create a PR to remove #18325 changes and hold till kettle redeployed and verified not uploading Garbage

### CODE REVIEW

File: kettle/make_db.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def pad_numbers(string):
     return re.sub(r'\d+', lambda m: m.group(0).rjust(16, '0'), string)
 
 WORKER_CLIENT = None  # used for multiprocessing
+GARBAGE_JOBS = ('pr-e2e-gce', 'maintenance-ci-testgrid-config-upload', 'ci-test-infra-benchmark-demo', 'ci-kubernetes-coverage-unit')
```
Comment (cjwagner) [inline kettle/make_db.py:42]: `GARBAGE_JOBS` 😢

Comment (MushuEE) [reply kettle/make_db.py:42]: I am happy to change to "Excluded Jobs" if we want job moral to stay high

File: kettle/make_db.py [hunk 2]
```diff
@@ -39,6 +39,12 @@ def pad_numbers(string):
     return re.sub(r'\d+', lambda m: m.group(0).rjust(16, '0'), string)
 
 WORKER_CLIENT = None  # used for multiprocessing
+GARBAGE_JOBS = (
+    'pr-e2e-gce',
+    'maintenance-ci-testgrid-config-upload',
```
Comment (BenTheElder) [inline kettle/make_db.py:44]: uhhh :this-is-fine:

File: kettle/make_db.py [hunk 3]
```diff
@@ -39,6 +39,12 @@ def pad_numbers(string):
     return re.sub(r'\d+', lambda m: m.group(0).rjust(16, '0'), string)
 
 WORKER_CLIENT = None  # used for multiprocessing
+GARBAGE_JOBS = (
+    'pr-e2e-gce',
```
Comment (BenTheElder) [inline kettle/make_db.py:43]: what even is this?

Comment (BenTheElder) [reply kettle/make_db.py:43]: I think we can remove this https://github.com/kubernetes/test-infra/search?q=pr-e2e-gce&unscoped_q=pr-e2e-gce

File: kettle/make_db.py (hunk 1 above)
Comment (cjwagner) [reply kettle/make_db.py]: I was just joking, it doesn't matter to me 😄

File: kettle/make_db.py (hunk 3 above)
Comment (MushuEE) [reply kettle/make_db.py:43]: Both of the Garbage Garbage jobs removed.

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**